docs/release-notes/2.38.1.md

```diff
@@ -0,0 +1,18 @@
+# Percona Monitoring and Management 2.38.1
+
+| **Release date:** | July 12, 2023                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open-source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+## Fixed issue
+
+[PMM-12293](https://jira.percona.com/browse/PMM-12293) - The upgrade to Grafana  9.2.20 in PMM 2.38 fixed critical security [CVE-2023-3128](https://grafana.com/blog/2023/06/22/grafana-security-release-for-cve-2023-3128/) by changing the authentication mechanism to rely solely on the user's unique ID provided by the identity provider.
+
+This change caused an authentication issue for second and subsequent connections to PMM using a Percona Account. With this PMM update, we are fixing this authentication issue to ensure that the "User already exists" error is no longer incorrectly displayed when users log back into PMM.
+
```

Comment:
But "User already exists"  was a "proper" error. The Core problem for the user - They can't Log in to the PMM at all. The error message they see is secondary.